src/lib/Algoan.enum.ts

```diff
@@ -202,3 +202,27 @@ export type LoanAccountType =
   | 'STUDENT'
   | 'MILITARY'
   | 'SMB';
+
+/**
+ * ScoresTypes only value
+ */
+export type ScoresType = 'CREDIT';
+
+/**
+ * AnalysisReliability enum
+ */
+export enum AnalysisReliability {
+  HIGH = 'HIGH',
+  MEDIUM = 'MEDIUM',
+  LOW = 'LOW',
+}
+
+/**
+ * RegularCashFlow configurations
+ */
+export enum RegularCashFlowConfiguration {
+  A = 'A',
+  B = 'B',
+  C = 'C',
+  D = 'D',
+}
```

Comment:
To avoid a breaking change, use a `type`

```suggestion
export type RegularCashFlowConfiguration = 'A' | 'B' | 'C' | 'D';
```